frontend/src/scenes/feature-flags/FeatureFlagInstructions.tsx

```diff
@@ -299,44 +190,86 @@ export function CodeInstructions({
+                                title={
+                                    <>
+                                        {`Feature flag payloads is only available in these libraries: ${PAYLOAD_OPTIONS.map(
+                                            (payloadOption) => payloadOption.value
+                                        )}`}
+                                    </>
+                                }
+                            >
+                                <IconInfo className="text-xl text-muted-alt shrink-0" />
+                            </Tooltip>
+                        </div>
+                        <>
                             <div className="flex items-center gap-1">
                                 <LemonCheckbox
-                                    label="Show payload option"
-                                    onChange={() => setShowPayloadCode(!showPayloadCode)}
+                                    label="Show bootstrap option"
+                                    checked={showBootstrapCode}
+                                    onChange={() => setShowBootstrapCode(!showBootstrapCode)}
+                                    disabled={
+                                        !BOOTSTRAPPING_OPTIONS.map((bo) => bo.value).includes(selectedOption.value) ||
+                                        !!featureFlag?.ensure_experience_continuity
+                                    }
                                 />
-                                <Tooltip
-                                    title={
-                                        <>
-                                            {`Feature flag payloads is only available in these libraries: ${PAYLOAD_OPTIONS.map(
-                                                (payloadOption) => payloadOption.value
-                                            )}`}
-                                        </>
+                                <Tooltip title={'Bootstrapping is only available in JavaScript and ReactNative.'}>
+                                    <IconInfo className="text-xl text-muted-alt shrink-0" />
+                                </Tooltip>
+                            </div>
+                            <div className="flex items-center gap-1">
+                                <LemonCheckbox
+                                    label="Show local evaluation option"
+                                    checked={showLocalEvalCode}
+                                    onChange={() => setShowLocalEvalCode(!showLocalEvalCode)}
+                                    disabled={
+                                        selectedOption.type !== LibraryType.Server ||
+                                        !!featureFlag?.ensure_experience_continuity
                                     }
-                                >
+                                />
+                                <Tooltip title={'Local evaluation is only available in server side libraries.'}>
```

Comment:
```suggestion
                                <Tooltip title={'Local evaluation is only available in server side libraries when persisting flags across auth steps is disabled.'}>
```

Reply:
too much text to add this to the bootstrapping and local eval tooltip, how can we make this easy to understand without more reading?

Reply:
We definitely need it in there somehow 😅 .

Doesn't need to go into the bootstrapping tooltip (I've recently found people who want bootstrap with persisting flags, and it works because they don't have latency requirements 🤷 ).

For local eval, how about something like:

`Local evaluation is only available in server side libraries`.

And when the persist flag across auth steps is checked:

`Local evaluation is incompatible with persisting flag across authentication steps.`

So, the tooltip is small, but changes based on what language/config is selected.